cmake/find_pip_cuda.py

```
@@ -0,0 +1,103 @@
+
+def _ensure_cuda_stub(cu_dir):
+    """Create a minimal libcuda.so stub for build-time -lcuda linking."""
+    stubs_dir = cu_dir / "lib" / "stubs"
+    stub = stubs_dir / "libcuda.so"
+    if stub.exists():
+        return
+    stubs_dir.mkdir(parents=True, exist_ok=True)
+    src = stubs_dir / "_stub.c"
+    try:
+        src.write_text("void cuGetErrorString(void){}\n")
+        subprocess.check_call(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)